Library/Edge.cs

```diff
@@ -23,6 +23,10 @@ namespace BMeshLib
      * next one, so the function Next() is provided to return either next1 or
      * next2 depending on the vertex of interest.
      */
+    /// <summary>
+    /// Links two <see cref="Vertex"/>s together, and may or may not be part of a <see cref="Face"/>.
+    /// </summary>
+    /// <remarks>Multiple <see cref="Face"/>s can share the same <see cref="Edge"/>.</remarks>
```

Comment:
> The technical note above is worth including as well, as a `<remarks>`
I think it would be better to not to, the end user doesn't really need to know how it works internally. As long as the API is clear how _it_ works that is all the user should need to care about.
Also worth noting that XML comments don't really support much in the way of formatting, so it would be rather unsightly.
Maybe an abridged version would be better to include?